app/models/employee.rb

```diff
@@ -15,4 +15,6 @@ class Employee < ActiveRecord::Base
   has_many :desired_skills
   has_many :skills, :through => :current_skills
   has_many :goals, :through => :desired_skills, :source => :skill
+  has_secure_password
```

Comment:
Isn't this automatically defined by Rails as an included method which allows access to other methods (authenticate, valid, etc)? I had to add the password_digest attribute to my model, but I'm not sure what further definition I need to provide?